pkgs/top-level/python-packages.nix

```diff
@@ -7802,7 +7802,10 @@ in {
 
   toggl-cli = callPackage ../development/python-modules/toggl-cli { };
 
-  tokenizers = disabledIf (!isPy3k) (toPythonModule (callPackage ../development/python-modules/tokenizers { }));
+  tokenizers = disabledIf (!isPy3k) (toPythonModule (callPackage ../development/python-modules/tokenizers {
+    inherit (pkgs.rustPlatform) cargoSetupHook fetchCargoTarball;
```

Comment:
Maybe we want to move them so that we can just add the both to the inputs of any python package.

Reply:
I removed this in favor of using `rustPlatform` in the derivation itself.